README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-    <img src="https://raw.githubusercontent.com/neptune-ai/neptune-client/assets/readme/Github-cover.png" width="1500" />
+    <img src="https://neptune.ai/wp-content/uploads/2024/07/Facebook-cover-image.png" width="1500" />
```

Comment:
Commit to orphaned `assets` branch and use github permalink :wink:

Reply:
I mean the imagefile

Reply:
`assets` is also a protected branch :/

Reply:
So make a small PR :wink:

Reply:
https://github.com/neptune-ai/neptune-client/pull/1849